app/models/upvs/form.rb

```diff
@@ -13,18 +13,18 @@ class Upvs::Form < ApplicationRecord
   has_many :related_documents, class_name: 'Upvs::FormRelatedDocument', foreign_key: 'upvs_form_id'
 
   def xslt_html
-    related_document('CLS_F_XSLT_HTML')
+    related_document('CLS_F_XSLT_HTML') || ::Upvs::FormRelatedDocumentsDownloader.new(self).download_related_document_by_type(:xslt_html)&.data
```

Comment:
Hmm, toto sa mi nezda. Toto stahuje lazy sposobom ci ako to funguje?

Reply:
Upravene, natiahne sa to hned pri stahovani spravy